src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -56,4 +61,24 @@ void setConversationStore(ConversationStore conversationStore) {
     void setMessageStore(MessageStore messageStore) {
         this.messageStore = messageStore;
     }
+
+    /**
+     * This function fires when a user navigates to the activity feed page.
+     * It then forwards to activityfeed.jsp for rendering.
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException, ServletException {
+
+        List<Conversation> conversations = conversationStore.getAllConversations();
+        List<User> users = userStore.getUsers();
+        List<Message> messages = messageStore.getMessages();
+//        for(User u: users){
```

Comment:
Delete commented out code. If it's necessary, it wouldn't be commented out. If it's not necessary, you don't need it anyway

Reply:
Done